akka-testkit/src/main/resources/reference.conf

```diff
@@ -26,8 +26,18 @@ akka {
       type = akka.testkit.CallingThreadDispatcherConfigurator
     }
   }
-  
-  actor.serialization-bindings {
-    "akka.testkit.JavaSerializable" = java
+
+  actor {
+    serializers {
+      test-message-serializer = "akka.testkit.TestMessageSerializer"
+    }
+
+    serialization-identifiers {
+      "akka.testkit.TestMessageSerializer" = 23
+    }
+
+    serialization-bindings {
+      "akka.testkit.JavaSerializable" = java
```

Comment:
In the long run we should remove this, but for now it will break all the test things, so I don't want to touch it.